src/ms.tex

```diff
@@ -203,8 +203,54 @@ \subsection{Project governance} \label{sec:project-governance}
 
 \secauthor{Erik Tollerud}
 
-Briefly summarize new procedures and governance structure, new CoCo, election
-process overview, etc.
+%Briefly summarize new procedures and governance structure, new CoCo, election
+%process overview, etc.
+
+As part of the process of developing Astropy into a long-term sustainable
+product, and to improve transparency and accountability, the Project agreed to
+write down and formalize our governance structure (partly supported by explicit
+funding for this purpose - see \S \ref{sec:project-funding}). At the 2019
+coordination meeting input was gathered from participants on what governance
+structures existed in the associated Open Source Software communities, and what
+would fit well with the needs of Astropy. This led into a ``retreat'' planned
+for March 2020, but due to the COVID-19 pandemic, this became a serious of
+virtual meetings of the ``Astropy Governance Working Group''.  This group
+drafted the APE0 document \citep{ape0}, which was then eventually ratified and
```

Comment:
I think there is a space? Did we ever set that standard? Same comment would apply to all similar occurrences. 
```suggestion
drafted the APE 0 document \citep{ape0}, which was then eventually ratified and
```